client/packages/inventory/src/Stocktake/DetailView/modal/StocktakeLineEdit/utils.ts

```diff
@@ -29,7 +29,7 @@ export const DraftLine = {
       id: FnUtils.generateUUID(),
       expiryDate: null,
       itemId: item.id,
-      sellPricePerPack: 0,
+      sellPricePerPack: item.itemStoreJoin?.defaultSellPricePerPack ?? 0,
```

Comment:
What is expected behaviour when adding new batch via stocktake?

- Select item
- Click `+ batch`
- Auto-populates the default pack size, and now the default sell price on the other tab
- I edit the pack size to something else
- Currently, sell price is still set, should it clear?

Reply:
Yes, mark-prins probably should be recalculated 😜 

We should clear the sell price; it's the default sell price of the default pack size and we're filling it in as a little help. if the user changes the pack size from the default, it's reasonable to expect them to enter a new price